man/systemd-sleep.conf.xml

```diff
@@ -195,6 +195,24 @@
         <xi:include href="version-info.xml" xpointer="v203"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>SleepMemMode=</varname></term>
+
+        <listitem><para>The string to be written to <filename>/sys/power/mem_sleep</filename>
+        when <option>SuspendState=mem</option> or <command>hybrid-sleep</command> is used.
+        More than one value can be specified by separating multiple values with whitespace. They will be
+        tried in turn, until one is written without error. If none of the writes succeed, the operation will
+        be aborted. Defaults to empty, i.e. the kernel default or kernel command line option
+        <varname>mem_sleep_default=</varname> is respected.</para>
```

Comment:
this stuff is all so cryptic. we should probably start listing common values here at least,